templates/ssl_ratios_raw_v1.sql

```diff
@@ -0,0 +1,24 @@
+SELECT
+  submission_date,
+  os,
+  country,
+  SUM(udf_get_key(histogram_parent_http_pageload_is_ssl, 0)) AS non_ssl_loads,
+  SUM(udf_get_key(histogram_parent_http_pageload_is_ssl, 1)) AS ssl_loads,
```

Comment:
using `SUM` here ensures that we don't get `NULL`s, which is important because `1 + NULL` is `NULL` not `1`